lms/djangoapps/lti_provider/models.py

```diff
@@ -12,3 +15,27 @@ class LtiConsumer(models.Model):
     """
     key = models.CharField(max_length=32, unique=True, db_index=True)
     secret = models.CharField(max_length=32, unique=True)
+
+
+@receiver(SCORE_CHANGED)
+def score_changed_handler(sender, **kwargs):  # pylint: disable=unused-argument
+    """
+    Consume signals that indicate score changes.
+
+    TODO: This function is a placeholder for integration with the LTI 1.1
+    outcome service, which will follow in a separate change.
+    """
+    message = """LTI Provider got score change event:
+        points_possible: {}
+        points_earned: {}
+        user: {}
+        course_id: {}
+        usage_id: {}
+    """
+    print message.format(
+        str(kwargs.get('points_possible', None)),
```

Comment:
you definitely don't need to cast to `str` here. `.format` is wonderful because it formats the item nicely to fit within a string. Using a `str()` cast here just makes your code unnecessarily unreadable. See https://docs.python.org/2/library/stdtypes.html#str.format